File: AprendendoConceitos/MelhorandoLógicaDeProgramaçãoEmDiferentesLinguagens/Desafio JS/Desafio.js
const valorSalario = parseFloat(prompt("Informe o valor do salário:"));
const valorBeneficios = parseFloat(prompt("Informe o valor dos benefícios:"));

const valorImposto = calcularImposto(valorSalario);
const saida = valorSalario - valorImposto + valorBeneficios;
console.log(saida.toFixed(2));

function calcularImposto(salario) {
  let aliquota;
  if (salario >= 0 && salario <= 1100) {
    aliquota = 0.05;
  } else if (salario > 1100.01 && salario <= 2500.00) {
    aliquota = 0.10;
  } else {
    aliquota = 0.15;
  }
  
  return aliquota * salario;
}

File: AprendendoConceitos/MelhorandoLógicaDeProgramaçãoEmDiferentesLinguagens/Desafio C#/desafio.cs
using System;

class desafio 
{
  public static void Main (string[] args) 
  {
    Console.Write("Digite o valor do salário: ");
    float valorSalario = float.Parse(Console.ReadLine());
    Console.Write("Digite o valor do benefício: ");
    float valorBeneficio = float.Parse(Console.ReadLine());

    float valorImposto = 0;
        
    if (valorSalario >= 0 && valorSalario <= 1100)
    { 
      valorImposto = 0.05F * valorSalario;
    }
    else if(valorSalario >= 1100.01 && valorSalario <= 2500.00)
    {
      valorImposto = 0.10F * valorSalario;
    }
    else
    {
      valorImposto = 0.15F * valorSalario;
    }
    float saida = valorSalario - valorImposto + valorBeneficio;
    Console.WriteLine(saida.ToString("0.00"));
  }
}